create_coda_files/create_coda_files.py

```diff
@@ -1,55 +1,153 @@
 import argparse
 import os
 import time
-import random
+from os import path
+import json
 
-from core_data_modules.traced_data.io import TracedDataJsonIO, TracedDataCodaIO
+from core_data_modules.cleaners import somali, Codes
+from core_data_modules.traced_data import Metadata, TracedData
+from core_data_modules.traced_data.io import TracedDataJsonIO, TracedDataCoda2IO
 from core_data_modules.util import IOUtils
+from core_data_modules.cleaners.cleaning_utils import CleaningUtils
+from core_data_modules.data_models import Scheme
 
-if __name__ == "__main__":
-    parser = argparse.ArgumentParser(description="Takes a TracedData json file and converts it to a CODA file")
-    parser.add_argument("user", help="User launching this program")
-    parser.add_argument("json_input_path", metavar="json-input-path",
-                        help="Path to the input JSON file, containing a list of serialized TracedData objects")
-    parser.add_argument("flow_name", metavar="flow-name",
-                        help="Name of activation flow from which this data was derived")
-    parser.add_argument("variable_name", metavar="variable-name",
-                        help="Name of message variable in flow")
-    parser.add_argument("coda_output_path", metavar="coda-output-path",
-                        help="Path to a Coda file to write processed messages to")
-    parser.add_argument("prev_coda_path", metavar="prev-coda-path",
-                        help="Path to a Coda file containing previously coded messages")
+# opens a coding scheme
+def _open_scheme(filepath):
+    with open(filepath, "r") as f:
+        firebase_map = json.load(f)
+        return Scheme.from_firebase_map(firebase_map)
+
+#Autocleaners that exist for this project
+AUTO_CLEANERS ={'gender':somali.DemographicCleaner.clean_gender,
```

Comment:
space around equals

Reply:
also you're being inconsistent about whether brackets have a line to themselves or not

Reply:
80 characters is increasingly feeling way too short. Also, I don't think it's worth religiously sticking to 80 when the difference is one additional character

Reply:
Got it